configs/remote-ign/pin-trustee.bu

```diff
@@ -0,0 +1,18 @@
+variant: fcos
+version: 1.6.0
+storage:
+  luks:
+    - name: root
+      label: luks-root
+      device: /dev/disk/by-partlabel/root
+      clevis:
+        custom:
+          needs_network: true
+          pin: trustee
+          config: '{"servers":[{"url":"http://192.168.50.1:8080","cert":""}],"path":"conf-cluster/12345/root"}'
```

Comment:
> `conf-cluster/12345/root`

Base repository can only be `default` (instead of `conf-cluster` here) as per [trustee-operator const](https://github.com/confidential-containers/trustee-operator/blob/85be112d8d1b4cdc7c2193532d87f03d13a03f51/internal/controller/common.go#L47)